internal/backend/db/dbgorm/gorm_test.go

```diff
@@ -239,23 +253,23 @@ func assertDeleted[T any](t *testing.T, f *dbFixture, m T) {
 
 var (
 	// testSessionID    = "ses_00000000000000000000000001"
-	testBuildID = "bld_00000000000000000000000001"
+	testBuildID = kittehs.Must1(uuid.FromBytes([]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}))
```

Comment:
let's define this as const?

Reply:
didn't change the behavior from before

Reply:
before there was prefix in each ID. Now it's const bytes. I suggest to define those const bytes as a const var

Reply:
this is not constable (can't be defined const)

```go
kittehs.Must1(uuid.FromBytes([]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}))
```